infra/main.bicep

```diff
@@ -135,6 +135,29 @@ param clientAppSecret string = ''
 // Used for optional CORS support for alternate frontends
 param allowedOrigin string = '' // should start with https://, shouldn't end with a /
 
+@description('IP or Subnet that is allowed access via the public network. Expectation is that the format is a comma-delimited list of CIDR IP rules')
```

Comment:
If you haven't yet, can you run az bicep format on the changed Bicep files to see if it fixes anything? (And yeah it doesnt do newlines right, so revert any newline at EOF file changes)